retrieve-digests/pkg/repository_handler.go

```diff
@@ -0,0 +1,76 @@
+package pkg
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+)
+
+// ProcessRepositories processes images for two registries and writes digests to CSV files
+func ProcessRepositories(ctx context.Context, registry1, registry2, project1, project2 string, repoFetcher RepositoryFetcher, writer1, writer2 *CSVWriter) (int, int, error) {
```

Comment:
The output could be clear adding the output name and type instead of only type. WDYT?

Reply:
```suggestion
func ProcessRepositories(ctx context.Context, registry1, registry2, project1, project2 string, repoFetcher RepositoryFetcher, writer1, writer2 *CSVWriter) (myfieldOuput1 int, myfieldOuput2 int, myfieldOuput3 error) {
```
In order to make it more readable ... but it's just a suggestion ok?